terratex_reallife/SYSTEM/scheinesys_vehicle.lua

```diff
@@ -169,6 +169,17 @@ function isPlane(vehid)
     end
 end
 
+function isWaterPlane(vehid)
+    if type(vehid) == "userdata" then
```

Comment:
Wozu diese prüfung? Call is doch eh nur mit id? Aber wenn du es tust kann man es so machen aber ich hätte dann dennoch gegen gecheckt ob es sich überhaupt um ein fahrzeug handelt um collisionen mit anderen ids zu verhindern

Reply:
Es geht mir hauptsächlich darum, dass das Wasserflugzeug nicht mehr auf Flughafen spawnt, sondern wie das Boot auf dem Wasser..